content/_data/changelogs/lts.yml

```diff
@@ -4785,6 +4785,377 @@
     message: |-
       Don't tell user's to signup when signup is not available.
 
+- version: "2.277.1"
+  date: 2021-03-10
+  lts_predecessor: "2.263.4"
+  lts_baseline: "2.277"
+  banner: >
+    As described in the <a href="/blog/2020/11/10/major-changes-in-weekly-releases/">Major changes in the weekly release line</a> blog post, this release improves key user interface components.
+    Configuration user interfaces now use html <code>div</code> markup to present a better layout than the previous html <code>table</code> layout.
+    Configuration pages are now easier to read, easier to understand, and work better on a wide range of screens.
+    The jQuery library used inside Jenkins has been upgraded from 2.1.4 to 3.5.1.
+    <p>&nbsp;</p>
+    <p>As described in the <a href="/blog/2020/11/10/spring-xstream/">Spring and XStream updates (breaking changes!)</a> blog post, this release updates and replaces outdated internal components.
+    The Acegi security library used for authentication has been replaced by Spring Security (<a href="https://github.com/jenkinsci/jep/blob/master/jep/227/README.adoc">JEP-227</a>).
+    A fork of the XStream library used to read and write XML files has been replaced by the upstream version of XStream (<a href="https://github.com/jenkinsci/jep/blob/master/jep/228/README.adoc">JEP-228</a>).
+
+  changes: # compared to lts_baseline 2.277 - extracted from the RC commit(s)
+
+  - type: security
+    pull: 5285
+    authors:
+    - dependabot
+    references:
+      - url: /security/advisory/2021-02-19/
+        title: security advisory
+      - url: https://github.com/spring-projects/spring-security/releases/tag/5.4.4
+        title: Spring Security 5.4.4 release notes
+      - url: https://github.com/spring-projects/spring-security/releases/tag/5.4.3
+        title: Spring Security 5.4.3 release notes
+    message: |-
+      Important security fix.
+      This vulnerability does not exist in any Jenkins LTS release.
+      It was discovered in a weekly release and fixed in a weekly release without being part of an LTS release.
```

Comment:
> Current text will be confusing in the case of backport

Can you explain further what you mean by "backport"?

The vulnerability exists in Jenkins versions 2.266 through 2.279 (inclusive) per the [security advisory](https://www.jenkins.io/security/advisory/2021-02-19/).  The change described by this entry is a change from 2.277 (vulnerable) to 2.277.1 (vulnerability fixed).  The vulnerability does not exist in Jenkins 2.263.4 because it uses the acegi security framework.

I'm also fine with keeping the phrasing simple, though if we want the simplest phrasing, I think we could copy "Important security fix" from the [2.280 changelog](https://www.jenkins.io/changelog/#v2.280) and rely on careful readers to realize that the Jenkins 2.277.1 changelog intentionally describes a security issue that was never included in any LTS release.  

I placed the extra sentences in hopes of reducing concern from 2.263.4 users that they might mistakenly assume that they must upgrade to 2.277.1 because there is a vulnerability in 2.263.4.

Reply:
"This vulnerability does not exist in any Jenkins LTS release." is a strange phrase for an LTS changelog (fine for weekly)

Reply:
I agree it is a surprising phrase for an LTS changelog.  I initially proposed to not mention this change but @daniel-beck felt that it was important to include it because it resolves a known security issue that would have arrived in 2.277.1 if this had not been backported.

I'm open to most anything in hopes of making it clear to 2.277.1 users that the vulnerability in 2.277 does not affect them while still making it clear that 2.263.4 is not affected by the security vulnerability that was introduced in 2.266.